app.py

```diff
@@ -550,11 +836,19 @@ async def send_chat_request(request):
 
     return response
 
+
 async def complete_chat_request(request_body):
-    response = await send_chat_request(request_body)
-    history_metadata = request_body.get("history_metadata", {})
+    if DATASOURCE_TYPE == "Promptflow":
```

Comment:
unlikely, we would use any other data sources other than promptflow. since all the connections and logic is embedded part of the promptflow.
i like the idea of using USE_PROMPTFLOW. let me update PR